samples/subsys/usb/audio/headphones_microphone/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&usbd {
+	hp_0 {
+		compatible = "usb-audio-hp";
+		feature-mute;
+		channel-l;
+		channel-r;
+		resolution = <16>;
+		ep-addr = <0x08>;
```

Comment:
Please remove `ep-addr` and `ep-sync-type`, they should not be there and could lead to assumption it can be configured.

Reply:
When it comes to ep-addr I will remove it and make it work with AUTO_EP_IN/OUT.
When it comes to ep-sync-type it actually can be configured, it's information to HOST how to synchronize the clocks.
Anyway I will delete this as Synchronization type Asynchronous for OUT endpoint requires feedback ISO IN endpoint and we will not support this for now.

Reply:
The default configuration will remain Synchronous as current descriptor structure supports this configuration.

Reply:
@jfischer-phytec-iot updated.